sim/.scalafmt.conf

```diff
@@ -0,0 +1,269 @@
+    "glob:**midas/targetutils/src/main/scala/midas/xdc/RAMStyleHint.scala",
+    "glob:**midas/targetutils/src/main/scala/midas/xdc/XDCAnnotation.scala",
+    "glob:**midas/targetutils/src/test/scala/ElaborationUtils.scala",
+    "glob:**midas/targetutils/src/test/scala/FpgaDebugSpec.scala",
+    "glob:**midas/targetutils/src/test/scala/PerfCounterSpec.scala",
+    "glob:**midas/targetutils/src/test/scala/RAMStyleHintSpec.scala",
+    "glob:**src/main/scala/fasedtests/AXI4Fuzzer.scala",
+    "glob:**src/main/scala/fasedtests/Config.scala",
+    "glob:**src/main/scala/midasexamples/AssertModule.scala",
+    "glob:**src/main/scala/midasexamples/AssertTorture.scala",
+    "glob:**src/main/scala/midasexamples/AutoCounter32bRollover.scala",
+    "glob:**src/main/scala/midasexamples/AutoCounterModule.scala",
+    "glob:**src/main/scala/midasexamples/AutoCounterValidator.scala",
+    "glob:**src/main/scala/midasexamples/Config.scala",
+    "glob:**src/main/scala/midasexamples/EnableShiftRegister.scala",
+    "glob:**src/main/scala/midasexamples/GCD.scala",
+    "glob:**src/main/scala/midasexamples/GlobalResetConditionTests.scala",
+    "glob:**src/main/scala/midasexamples/InstanceNameHelper.scala",
+    "glob:**src/main/scala/midasexamples/MultiReg.scala",
+    "glob:**src/main/scala/midasexamples/MultiRegfile.scala",
+    "glob:**src/main/scala/midasexamples/MultiSRAM.scala",
+    "glob:**src/main/scala/midasexamples/MulticlockAutoCounterModule.scala",
+    "glob:**src/main/scala/midasexamples/MulticlockPrintfModule.scala",
+    "glob:**src/main/scala/midasexamples/NestedModels.scala",
+    "glob:**src/main/scala/midasexamples/Parity.scala",
+    "glob:**src/main/scala/midasexamples/PassthroughModels.scala",
+    "glob:**src/main/scala/midasexamples/PointerChaser.scala",
+    "glob:**src/main/scala/midasexamples/PrintfModule.scala",
+    "glob:**src/main/scala/midasexamples/Regfile.scala",
+    "glob:**src/main/scala/midasexamples/ResetPulseBridgeTest.scala",
+    "glob:**src/main/scala/midasexamples/ResetShiftRegister.scala",
+    "glob:**src/main/scala/midasexamples/Risc.scala",
+    "glob:**src/main/scala/midasexamples/RiscSRAM.scala",
+    "glob:**src/main/scala/midasexamples/ShiftRegister.scala",
+    "glob:**src/main/scala/midasexamples/Stack.scala",
+    "glob:**src/main/scala/midasexamples/Termination.scala",
+    "glob:**src/main/scala/midasexamples/TriggerPredicatedPrintf.scala",
+    "glob:**src/main/scala/midasexamples/TriggerWiringModule.scala",
+    "glob:**src/main/scala/midasexamples/TrivialMulticlock.scala",
+    "glob:**src/main/scala/midasexamples/TwoAdders.scala",
+    "glob:**src/main/scala/midasexamples/Util.scala",
+    "glob:**src/main/scala/midasexamples/VerilogAccumulator.scala",
+    "glob:**src/main/scala/midasexamples/WireInterconnect.scala",
+    "glob:**src/test/scala/fasedtests/FASEDTestSuite.scala",
+    "glob:**src/test/scala/midasexamples/TutorialSuite.scala"
+  ]
+}
+
+maxColumn = 120
```

Comment:
I know people like smaller values of this... but I don't think this is an issue.